lib/elixir/test/elixir/gen_server_test.exs

```diff
@@ -50,8 +55,17 @@ defmodule GenServerTest do
     assert GenServer.cast(:foo, {:push, :world}) == :ok
   end
 
-  test "call/3 exit messages" do
+  test "call/3 exit messages when calling self" do
     assert catch_exit(GenServer.call(self, :pop, 5000)) == {:calling_self, {GenServer, :call, [self, :pop, 5000]}}
+
```

Comment:
Because if we use the same format as the first test, `GenServer.whereis(server)` would not equal `self()`.